src/util/pelz_io.c

```diff
@@ -127,37 +130,258 @@ int key_load(size_t key_id_len, unsigned char *key_id, size_t * key_len, unsigne
   case FILE_URI:
```

Comment:
I think the code will be easier to read if there are two separate key load functions depending on if it's a ```FILE_URI``` or ```PELZ_URI```.

Reply:
I do not disagree but I don't think is is right for this merge request.  I am working on a new branch to improve the readability of the load into enclave code which should help.